src/zrender.ts

```diff
@@ -126,7 +127,17 @@ class ZRender {
         const handerProxy = (!env.node && !env.worker && !ssrMode)
             ? new HandlerProxy(painter.getViewportRoot(), painter.root)
             : null;
-        this.handler = new Handler(storage, painter, handerProxy, painter.root);
+
+        const useCoarsePointer = opts.useCoarsePointer;
+        const usePointerSize = (useCoarsePointer == null || useCoarsePointer === 'auto')
+            ? isTouchDevice()
+            : !!useCoarsePointer;
+        const defaultPointerSize = 44;
+        const targetSize = usePointerSize
+            ? (opts.pointerSize == null ? defaultPointerSize : opts.pointerSize)
+            : 0;
```

Comment:
`0` looks not good semantically when not using the coarse pointer. I prefer `undefined/null`. 

And this block can be simplified as
```ts
let pointerSize;
if (usePointerSize) {
  pointerSize = retrive2(opts.pointerSize, defaultPointerSize);
}
// or
usePointerSize && (pointerSize = retrive2(opts.pointerSize, defaultPointerSize);
```

Optionally, `defaultPointerSize` can also be replaced with its value `44` and add a comment for it.